### PR TITLE
Feature new edit entries

### PR DESCRIPTION
**Pull request by tnagorra**

### CODE REVIEW

File: src/views/EditAry/RightPanel/Score/index.js [hunk 1]
```diff
@@ -152,7 +117,11 @@ export default class Score extends React.PureComponent {
                 <td className={styles.cell}>
                     <ScaleInput
                         faramElementName={String(id)}
-                        options={scaleValues}
+                        options={this.props.assessmentScoreScales}
+                        keySelector={option => option.value}
```
Comment (AdityaKhatri) [inline src/views/EditAry/RightPanel/Score/index.js]: No inline functions.

File: src/views/EditAry/RightPanel/Score/index.js [hunk 2]
```diff
@@ -248,8 +215,10 @@ export default class Score extends React.PureComponent {
                     faramElementName={String(sectorId)}
                     rows={pillarData.rows}
                     columns={pillarData.columns}
-                    scaleValues={scaleValues}
+                    scaleValues={this.props.assessmentScoreScales}
                     scales={pillarData.scales}
+                    keySelector={option => option.value}
```
Comment (AdityaKhatri) [inline src/views/EditAry/RightPanel/Score/index.js]: here as well

File: src/views/EditEntries/Overview/index.js [hunk 3]
```diff
@@ -0,0 +1,184 @@
...
+                    <React.Fragment>
+                        <header className={styles.header}>
+                            <SelectInput
+                                className={styles.entrySelectInput}
+                                hideClearButton
+                                keySelector={Overview.entryKeySelector}
+                                labelSelector={Overview.entryLabelSelector}
+                                onChange={this.handleEntrySelect}
+                                options={this.props.entries}
+                                placeholder="Select entry"
```
Comment (AdityaKhatri) [inline src/views/EditEntries/Overview/index.js]: Add string or fixme

File: src/views/EditEntries/Overview/index.js [hunk 4]
```diff
@@ -0,0 +1,184 @@
...
+                                showHintAndError={false}
+                                showLabel={false}
+                                value={this.props.selectedEntryKey}
+                            />
+                            <div className={styles.actionButtons}>
+                                <SuccessButton
+                                    onClick={this.handleEmptyExcerptCreate}
+                                >
+                                    Add entry
```
Comment (AdityaKhatri) [inline src/views/EditEntries/Overview/index.js]: Add string or fixme

File: src/views/EditEntries/Overview/index.js [hunk 5]
```diff
@@ -0,0 +1,184 @@
...
+                                </SuccessButton>
+                                <DangerButton
+                                    onClick={this.handleEntryDelete}
+                                    disabled={!entry}
+                                >
+                                    Remove entry
```
Comment (AdityaKhatri) [inline src/views/EditEntries/Overview/index.js]: Y u no add string?

File: src/views/EditEntries/WidgetFaram/index.js [hunk 6]
```diff
@@ -0,0 +1,178 @@
...
+    renderWidgetHeader = (widget) => {
+        const { title, id } = widget;
+
+        const {
+            actionComponent: ActionComponent,
+            entry,
+        } = this.props;
+
+        const {
+            data: { attributes: { [id]: { data } = {} } = {} } = {},
```
Comment (AdityaKhatri) [inline src/views/EditEntries/WidgetFaram/index.js:81]: Use defined emptyObject instead of {}

File: src/views/EditEntries/widgets/Date/index.js [hunk 7]
```diff
@@ -0,0 +1,31 @@
...
+
+    render() {
+        // TODO: feature to set date to published date automatically
+        return (
+            <DateInput
+                faramElementName="value"
+                faramInfo={{
+                    action: 'changeExcerpt',
+                    type: 'excerpt',
+                    value: 'This excerpt was changed as a side effect.',
```
Comment (AdityaKhatri) [inline src/views/EditEntries/widgets/Date/index.js]: Add fixme

File: src/views/EditEntries/WidgetFaram/index.js (hunk 6 above)
Comment (tnagorra) [reply src/views/EditEntries/WidgetFaram/index.js:81]: No need

Comment (AdityaKhatri) [reply src/views/EditEntries/WidgetFaram/index.js:81]: ok

**Pull request merged**